frontend/packages/core/src/card.tsx

```diff
@@ -51,23 +52,62 @@ const StyledCardHeaderContainer = styled.div({
   background: "#EBEDFB",
 });
 
+const StyledCardHeader = styled.div({
+  padding: "16px",
+  display: "flex",
+  alignItems: "center",
+  height: "68px",
+});
+
+const StyledCardHeaderAvater = styled.div({
+  marginRight: "16px",
+});
+
+// TODO: confirm the width
+const StyledDivider = styled(Divider)({
+  color: "rgba(13, 16, 48, 0.38)",
+  height: "36px",
+});
+
+const StyledGridItem = styled(Grid)({
+  textAlign: "center",
+});
+
+export interface CardHeaderSections {
+  title: React.ReactNode;
+  subheader?: React.ReactNode;
```

Comment:
We might want to make this more narrow for now since we don't want any element rendered. e.g. probably want this to only be text for now and it might be good to force consumers to use a Typography component instead of just a string.

Reply:
I changed the `subheader` to be a string type and use the Typography component.

For `title` - in figma, the title color can vary per section and internally, I'm passing in the `TimeAgo` component for some of the sections, so perhaps we keep this field flexible?